tests/unit_tests/test_utils.py

```diff
@@ -0,0 +1,45 @@
+import os
+
+from openmc import _utils
+import pytest
+
+# from tests import cdtemp
+
+
+# @pytest.fixture(scope="module")
+# def download_photos():
+@pytest.fixture()
+def download_photos(run_in_tmpdir):
+    """use _utils download() function to download the same picture four times,
+       twice to get unique names, & a third time to use the already downloaded
+       block of code, and a fourth time to use the checksum block"""
```

Comment:
This docstring looks like it needs to be updated (the image is downloaded 3 times, not 4)